src/main/java/com/auth0/AuthorizeUrl.java

```diff
@@ -41,6 +79,18 @@ public AuthorizeUrl withConnection(String connection) {
         return this;
     }
 
+    /**
+     * Sets whether a fallback cookie should be used for clients that do not support "SameSite=None".
+     * Only applicable when this instance is created with {@link AuthorizeUrl#AuthorizeUrl(AuthAPI, HttpServletRequest, HttpServletResponse, String, String)}.
+     *
+     * @param legacySameSiteCookie whether or not to set fallback auth cookies for clients that do not support "SameSite=None"
+     * @return the builder instance
+     */
+    AuthorizeUrl withLegacySameSiteCookie(boolean legacySameSiteCookie) {
```

Comment:
rename parameter `useLegacySameSiteCookie`? (check all usages!)

Reply:
Renamed all legacySameSiteCookie param names to `useLegacySameSiteCookie`.